layouts/joomla/html/batch/access.php

```diff
@@ -31,4 +24,4 @@
 			'title' => Text::_('JLIB_HTML_BATCH_NOCHANGE'),
 			'id'    => 'batch-access'
 		)
-	); ?>
+	);
```

Comment:
```suggestion
	); ?>
```
I think the `<?php echo HTMLHelper::_(` need a closing `?>` but this is removed here.

Reply:
Or doesn't it need that if it's the end of the file?

Reply:
I see that's the case, so forget my comment. Will mark as resolved.